w3d_translator/__init__.py

```diff
@@ -1,6 +1,6 @@
 __version__ = "0.1.0"
 
-from .read_xml import parse_string
+from .translate import parse_string
```

Comment:
I thought relative imports have been deprecated?

Reply:
I think they might still be okay – at least the documentation says they're fine (as we're inside a package) – or have I missed something? https://docs.python.org/3/reference/import.html#package-relative-imports

Reply:
It still works fine as `from translate import parse_string` so I'll make the change